commands/config.go

```diff
@@ -78,6 +178,69 @@ func configGetCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func configSetCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	config, response := c.GetConfig()
+	if response.Error != nil {
+		return response.Error
+	}
+
+	path, err := parseConfigPath(args[0])
+	if err != nil {
+		return err
+	}
+	if err := setConfigValue(path, config, args[1:]); err != nil {
+		return err
+	}
+	if _, res := c.UpdateConfig(config); res.Error != nil {
+		return res.Error
+	}
+
+	printer.Print("Value changed successfully")
+	return nil
+}
+
+func configResetCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	confirmFlag, _ := cmd.Flags().GetBool("confirm")
+
+	if !confirmFlag && len(args) > 0 {
+		var confirmResetAll string
+		confirmationMsg := fmt.Sprintf(
+			"Are you sure you want to reset %s to their default value? (YES/NO): ",
+			args[0])
+		printer.Print(confirmationMsg)
```

Comment:
Confirmation message should be printed directly with `fmt`, as we don't want it to be included as part of the JSON output and we do want it to be interactive (and `--format json` would accumulate it instead of printing it)